client/send-video.cc

```diff
@@ -260,6 +290,6 @@ int main(int argc, char *argv[]) {
 
     // Close the video file
     avformat_close_input(&pFormatCtx);
-
-    return 0;
+    fprintf(stderr, "Finished playing %s - %ld frames %ld times for %0.1fs\n", filename, frame_count, repeated_count + 1, (GetTimeInMillis() - startTime) / 1000.);
+    return false;
```

Comment:
Since interrupt_received is scoped outside this function I think it'd make the most sense to return something more meaningful about the result of playing. I've got an idea.